applications/PfemFluidDynamicsApplication/custom_processes/generate_new_nodes_before_meshing_process.hpp

```diff
@@ -2010,22 +2045,30 @@ namespace Kratos
 				}
 				else
 				{
-					// Master node's properties are set using the maximum PROPERTY_ID value between SlaveNode1 and SlaveNode2
-					if (SlaveNode1->FastGetSolutionStepValue(PROPERTY_ID) >= SlaveNode2->FastGetSolutionStepValue(PROPERTY_ID))
-					{
-						TakeMaterialPropertiesFromNotRigidNode(pnode, SlaveNode1);
-					}
-					else
-					{
-						TakeMaterialPropertiesFromNotRigidNode(pnode, SlaveNode2);
-					}
+
+					TakeMaterialPropertiesFromNotRigidNode(pnode, SlaveNode1);
+					// unsigned int propertyIdFirstNode = SlaveNode1->FastGetSolutionStepValue(PROPERTY_ID);
+					// unsigned int propertyIdSecondNode = SlaveNode2->FastGetSolutionStepValue(PROPERTY_ID);
+					// if (propertyIdFirstNode != propertyIdSecondNode && SlaveNode1->IsNot(RIGID) && SlaveNode2->IsNot(RIGID))
+					// {
+					// 	std::cout << "I SHOULD NOT ENTER HEEEEEEEEEEEEERE " << propertyIdFirstNode << " VS " << propertyIdSecondNode << std::endl;
```

Comment:
we need this commented code?

Reply:
I should have removed it with the last committ